packages/common/src/functions-api.ts

```diff
@@ -62,17 +65,23 @@ export interface GetDonorAppointmentsResponse {
   futureAppointments: BookedAppointmentApiEntry[];
 }
 
-export const BookAppointmentFunctionName = "bookAppointment";
+export const CoordinatorBookAppointmentFunctionName =
+  "coordinatorBookAppointment";
+export const DonorBookAppointmentFunctionName = "bookAppointment";
+export interface BookAppointmentRequest {
```

Comment:
Why are donors and coordinators use the same request object?
I think coordinators should have a separate request object for a separate function

Reply:
done